### DISCUSSION TITLE
Analog joystick

### DISCUSSION THREAD

**Discussion by rawsp33d** [JellyTitan/Sofle-Pico#19] [Ideas]
Let's say hypothetically I wanted to attach a controller analog stick to the PCB, probably right where the rotary encoders are. What I have in mind is a [GameCube controller stickbox](https://a.co/d/hOiO0RN). It is similar in form factor to the rotary encoders you used but the most notable difference is 6 pins instead of 5 (3 per potentiometer).

I saw you were playing around with other dare I say "modules" in that place such as pimoroni trackpads, and my mind immediately went there. If I could use this sucker to play Melee I would do so in a heartbeat. Not to mention controlling my mouse with one sounds slightly better than some other options (though an old school track ball module sounds best to me in that regard).

I bet it's technically very easy, I'm just wondering about how poka-yoke it might be.

Thanks!

**Comment by JellyTitan** [comment]
I haven't seen one of these incorporated into a QMK keyboard before. I would recommend searching the QMK_firmware repo for somebody that's tried it. If you can find something like that, we can crib the footprint and possibly the firmware config. It might be a pain in the butt to port it from pro micro to pico though. As far as adding that functionality from scratch, that's probably beyond me. I'm standing on the shoulders of Giants here.
